pkg/volume/glusterfs/glusterfs.go

```diff
@@ -346,7 +346,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 			return nil
 		}
 
-		const invalidOption = "Invalid option auto_unmount"
+		const invalidOption = "Invalid argument"
```

Comment:
To avoid breaking other version, could we check both strings "Invalid option auto_unmount" and "Invalid argument"?

Reply:
@jingxu97 Set it to "Invalid" to match any of both.